### ISSUE TITLE
Batch Normalization Layer

### ISSUE THREAD

**Issue by manuelelucchi** [team-reframe/Daedalus#19]
https://kevinzakka.github.io/2016/09/14/batch_normalization/

**Comment by manuelelucchi** [comment]
Done
